From 7099411ed66f42243b4bb89408ff7123097292b8 Mon Sep 17 00:00:00 2001
From: Ximin Luo <infinity0@pwned.gg>
Date: Sat, 4 Jan 2020 05:11:21 +0000
Subject: [PATCH] Add cross prefix for riscv64gc-unknown-linux-gnu =>
 riscv64-linux-gnu

Add out-of-the-box cross-compiling support for riscv64gc-unknown-linux-gnu
---
 src/lib.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib.rs b/src/lib.rs
index 726d1286..55c9b9d4 100644
--- a/vendor/cc/src/lib.rs
+++ b/vendor/cc/src/lib.rs
@@ -2027,6 +2027,7 @@ impl Build {
                         "riscv32imc-unknown-none-elf" => Some("riscv32-unknown-elf"),
                         "riscv64gc-unknown-none-elf" => Some("riscv64-unknown-elf"),
                         "riscv64imac-unknown-none-elf" => Some("riscv64-unknown-elf"),
+                        "riscv64gc-unknown-linux-gnu" => Some("riscv64-linux-gnu"),
                         "s390x-unknown-linux-gnu" => Some("s390x-linux-gnu"),
                         "sparc-unknown-linux-gnu" => Some("sparc-linux-gnu"),
                         "sparc64-unknown-linux-gnu" => Some("sparc64-linux-gnu"),
